packages/theme-chalk/src/statistic.scss

```diff
@@ -0,0 +1,52 @@
+@use 'mixins/mixins' as *;
+@use 'mixins/var' as *;
+@use 'common/var' as *;
+
+@include b(statistic) {
+  @include set-component-css-var('statistic', $statistic);
+}
+
+@include b(statistic) {
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 14px;
+  font-variant: tabular-nums;
+  line-height: 1.5715;
+  list-style: none;
+  font-feature-settings: 'tnum';
+
+  .head {
+    padding: 0 5px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    letter-spacing: -0.01px;
+    width: 100%;
+    display: inline-flex;
+    align-items: center;
+    justify-content:center ;
+    color: #606266;
+  }
+
+  .con {
+    display: flex;
+    justify-content: center;
+    align-items: center ;
+  padding-top: 8px;
+    .number {
+      font-style: normal;
+      padding: 0 5px;
+      font-family: sans-serif;
+    }
+    span {
```

Comment:
## Suggest

A more specific class name is required, and refactoring using bem mxins is recommended.

example

```
// steps.scss
@use 'mixins/mixins' as *;
@use 'common/var' as *;

@include b(steps) {
  display: flex;

  @include m(simple) {
    padding: 13px 8%;
    border-radius: 4px;
    background: getCssVar('fill-color', 'light');
  }

  @include m(horizontal) {
    white-space: nowrap;
  }

  @include m(vertical) {
    height: 100%;
    flex-flow: column;
  }
}
```